<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_char_form</class>
 <widget class="QWidget" name="create_char_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="char_form_title">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Character Form</string>
   </property>
  </widget>
  <widget class="QLabel" name="denotes_req_field_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* Denotes Required Field</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="backstory_text_val">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="back_strory_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back Story</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="apperance_text_val">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="apperance_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Apperance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>370</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="basic_info_groupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>231</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Basic Info</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <property name="verticalSpacing">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>17</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item row="9" column="0">
     <widget class="QLabel" name="eye_label">
      <property name="text">
       <string>Eye color</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="eye_val"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="age_label">
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="age_val"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="weight_label">
      <property name="text">
       <string>Weight</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="weight_val"/>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="Height_label">
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="height_val"/>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="skin_label">
      <property name="text">
       <string>Skin color</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QLineEdit" name="skin_val"/>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="hair_label">
      <property name="text">
       <string>Hair color</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLineEdit" name="hair_val"/>
    </item>
    <item row="21" column="0">
     <widget class="QLabel" name="Flaw_label">
      <property name="text">
       <string>Flaw</string>
      </property>
     </widget>
    </item>
    <item row="21" column="1">
     <widget class="QLineEdit" name="flaw_val"/>
    </item>
    <item row="23" column="0">
     <widget class="QLabel" name="personality_trait_label">
      <property name="text">
       <string>Personality Traits</string>
      </property>
     </widget>
    </item>
    <item row="23" column="1">
     <widget class="QLineEdit" name="personality_val"/>
    </item>
    <item row="25" column="0">
     <widget class="QLabel" name="bonds_label">
      <property name="text">
       <string>Bonds</string>
      </property>
     </widget>
    </item>
    <item row="25" column="1">
     <widget class="QLineEdit" name="bonds_val"/>
    </item>
    <item row="27" column="0">
     <widget class="QLabel" name="ideals_label">
      <property name="text">
       <string>Ideals</string>
      </property>
     </widget>
    </item>
    <item row="27" column="1">
     <widget class="QLineEdit" name="ideals_val"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="race_label">
      <property name="text">
       <string>Race</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="class_label">
      <property name="text">
       <string>Class</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="background_label">
      <property name="text">
       <string>Bakcground</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="name_val"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="name_label">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="race_box">
      <item>
       <property name="text">
        <string>Dragonborn</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dwarf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Elf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gnome</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half-elf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half-orc</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Halfling</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Human</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tiefling</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="class_box">
      <item>
       <property name="text">
        <string>Barbarian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bard</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cleric</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Druid</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fighter</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Monk</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Paladin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ranger</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rogue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sorcerer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Warlock</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wizard</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="background_box">
      <item>
       <property name="text">
        <string>Acolyte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Charlatan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Criminal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Entertainer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Folk Hero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Guild Artisan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hermit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Noble</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Outlander</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sailor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Soldier</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Urchin</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="attribute_groupbox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>40</y>
     <width>171</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Attributes</string>
   </property>
   <widget class="QLabel" name="required_astric_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="charisma">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="str_label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Strength</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dexterity">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>46</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="wisdom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>124</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cha_label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Charisma</string>
    </property>
   </widget>
   <widget class="QLabel" name="con_label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>72</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Constitution</string>
    </property>
   </widget>
   <widget class="QLabel" name="int_label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>98</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Intellegence</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="strength">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="wis_label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>124</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wisdom</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="intellegence">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>98</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dex_label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>46</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dexterity</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="constitution">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>72</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="alignment_box">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lawful Good</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neutral Good</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chaotic Good</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lawful Neutral</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neutral</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chaotic Neutral</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lawful Evil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neutral Evil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chaotic Evil</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="alignment_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Alignment</string>
   </property>
  </widget>
  <widget class="QLabel" name="languages_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string>Languages</string>
   </property>
  </widget>
  <widget class="QListWidget" name="language_list">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>256</width>
     <height>91</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <item>
    <property name="text">
     <string>Common</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dwarvish</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Elvish</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Giant</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gnomish</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Goblin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Halfling</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orc</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Abyssal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Celestial</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Draconic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Deep Speech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Infernal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Primordial</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sylvan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Undercommon</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="required_astric_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>25</y>
     <width>123</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
